pages/data-design/collaboration/members.mdx

```diff
@@ -6,7 +6,27 @@ Expand your Avo team and manage your members access by opening the workspace set
 
 ## Roles
 
-You can assign roles when inviting new users as well as for existing users, excluding yourself.
+Admins can assign roles when inviting new users as well as for existing users, excluding yourself.
+
+Role overview:
+
+|                                        | [**Billing**](/data-design/collaboration/members#billing-only) | [**Viewer**](/data-design/collaboration/members#viewer) | [**Editor**](/data-design/collaboration/members#editor) | [**Admin**](/data-design/collaboration/members#admin) |
+|----------------------------------------|---------|--------|--------|-------|
+| View tracking plan                     |         | ✅      | ✅      | ✅     |
+| Pull generated code using the Avo CLI  |         | ✅✳️     | ✅      | ✅     |
```

Comment:
Can we change this to a regular asterisk rather than the unicode/emoji?
```suggestion
| Pull generated code using the Avo CLI  |         | ✅*     | ✅      | ✅     |
```

Reply:
I started with that, but it comes out as a bullet when I add the asterisk below the table 🫠 Do you know how to undo the transformation into a bullet point?

Reply:
my buddy, google helped me out